src/main/webapp/repo-bootstrap/global/blueprints/empty/config/studio/administration/samples/sample-site-config.xml

```diff
@@ -4,6 +4,9 @@
     <display-name>{siteName}</display-name>
     <default-timezone>EST5EDT</default-timezone>
 
+    <!-- Server used for preview, if not specified it defaults to the same server where Studio is running. -->
+    <previewServer>http://localhost:3000</previewServer>
```

Comment:
Please comment out the non-default preview server in case someone decides to use the sample config.